appservice/src/createAppService/createWebApp.ts

```diff
@@ -23,17 +27,19 @@ export async function setWizardContextDefaults(wizardContext: IAppServiceWizardC
     await LocationListStep.setLocation(wizardContext, 'centralus');
     // defaults that for if one workspace is opened
     if (workspace.workspaceFolders && workspace.workspaceFolders.length === 1) {
-        await workspace.findFiles('package.json').then((files: Uri[]) => {
-            if (files.length > 0) {
-                wizardContext.newSiteOS = WebsiteOS.linux;
-            }
-        });
-
-        await workspace.findFiles('*.csproj').then((files: Uri[]) => {
-            if (files.length > 0) {
-                wizardContext.newSiteOS = WebsiteOS.windows;
-            }
-        });
+        const fsPath: string = workspace.workspaceFolders[0].uri.fsPath;
+        if (await fse.pathExists(path.join(fsPath, 'package.json'))) {
+            wizardContext.newSiteOS = WebsiteOS.linux;
+        } else if (await fse.pathExists(path.join(fsPath, 'requirements.txt'))) {
+            // requirements.txt are used to pip install so a good way to determine it's a Python app
+            wizardContext.newSiteOS = WebsiteOS.linux;
+            wizardContext.detectedSiteRuntime = pythonRuntime;
```

Comment:
Currently this is only set if `advancedCreation` is set to false. Since this just changes the ordering of the quick pick - shouldn't it be set even if `advancedCreation` is true?